packages/core_unix/core_unix.v0.16.0/opam

```diff
@@ -30,6 +30,7 @@ description: "
 Unix-specific extensions to some of the modules defined in [core] and [core_kernel].
 "
 depexts: ["linux-headers"] {os-family = "alpine"}
+available: [ !(os = "freebsd" & os-version >= "14") ]
```

Comment:
Does it mean that it works for freebsd < 14?

Reply:
If I understand the issue that @jmid, linked and @hannesm reported yes. The underlying issue seems to be that FreeBSD 14 started shipping `sys/timerfd.h` which trips up the detection in `core_unix`.

Reply:
Thats also how I understand https://github.com/ocurrent/ocaml-ci/issues/924#issuecomment-2008135595 :+1: